meta-ros-common/recipes-support/libyaml/libyaml_0.2.5.bb

```diff
@@ -0,0 +1,27 @@
+SUMMARY = "LibYAML is a YAML 1.1 parser and emitter written in C."
```

Comment:
there is already `openembedded-core/meta/recipes-support/libyaml/libyaml_0.2.5.bb`, why do we create our own?

Reply:
Good question.  Here is the explanation I included in the patch file:

Reply:
this bb package makes error below, and everythings gonna be ok after I remove `meta-ros-common/recipes-support/libyaml`

```
ERROR: python3-pyyaml-6.0-r0 do_package_qa: QA Issue: /usr/lib/python3.10/site-packages/yaml/_yaml.cpython-310-aarch64-linux-gnu.so contained in package python3-pyyaml requires libyaml-0.so.2()(64bit), but no providers found in RDEPENDS:python3-pyyaml? [file-rdeps]
ERROR: python3-pyyaml-6.0-r0 do_package_qa: Fatal QA errors were found, failing task.
```